### DISCUSSION TITLE
Can open-android-backup be the backup solution for the OpenAndroidInstaller?

### DISCUSSION THREAD

**Discussion by tsterbak** [mrrfv/open-android-backup#82] [Q&A]
Hey, I'm working on an automatic installer for custom ROMs (https://github.com/openandroidinstaller-dev/openandroidinstaller).
And this project would benefit greatly from a safe, reliable and automatic back-up solution built in.

Do you think open-android-backup could be that solution?
Is it ROM agnostic and allows restoring on different ROMs than the one the back-up was taken from?

**Comment by mrrfv** [comment]
Quoting my reply in another issue:

> Yes, Open Android Backup uses very standard methods of backing up / restoring data so it should be ROM-agnostic. Some ROMs however do not support the `tar` export method, so it might be better to use the `adb` option instead. Also, you can use [environment variables](https://github.com/mrrfv/open-android-backup#automationunattended-backups) to automatically change some backup settings - including the export method. Let me know if you have any more questions.
